core/src/main/java/org/apache/accumulo/core/security/ColumnVisibility.java

```diff
@@ -151,7 +151,7 @@ public int getTermEnd() {
 
     public ByteSequence getTerm(byte[] expression) {
       if (type != NodeType.TERM) {
-        throw new RuntimeException();
+        throw new IllegalStateException();
```

Comment:
```suggestion
        throw new IllegalStateException("unexpected node type" );
```

Reply:
Originally committed this, but reverted because this (or another one) broke the formatter, and my change is more minimal. This might be an improvement, but not one I'm inclined to bother with here. Maybe it'd be worth putting the message in a String variable and reusing it, but that's out of scope of my change, which is just using the more specific type.